docs/source/governance.rst

```diff
@@ -12,12 +12,17 @@ Leads
 Core Maintainers
 ----------------
 - Jirka Borovec (`Borda <https://github.com/Borda>`_)
+- Kushashwa Ravi Shrimali (`krshrimali` <https://github.com/krshrimali>`_)
 - Kaushik Bokka (`kaushikb11 <https://github.com/kaushikb11>`_)
 - Justus Schock (`justusschock <https://github.com/justusschock>`_)
-- Carlos Mocholí (`carmocca <https://github.com/carmocca>`_)
-- Sean Narenthiran (`SeanNaren <https://github.com/SeanNaren>`_)
 - Akihiro Nitta (`akihironitta <https://github.com/akihironitta>`_)
 - Aniket Maurya (`aniketmaurya <https://github.com/aniketmaurya>`_)
 - Ananya Harsh Jha (`ananyahjha93 <https://github.com/ananyahjha93>`_)
 - Sivaraman Karthik Rangasai (`karthikrangasai <https://github.com/karthikrangasai>`_)
 - Pietro Lesci (`pietrolesci <https://github.com/pietrolesci>`_)
+
+Alumni
+------
+
+- Carlos Mocholí (`carmocca <https://github.com/carmocca>`_)
```

Comment:
Hey Kush! I appreciate the detail!

This might signal that I left the team. I just wanted to remove myself from CODEOWNERS since I am not actively reviewing Flash.

So I think it's best i'm kept off here to avoid confusion

```suggestion
```